.github/workflows/ci-edge-functions.yml

```
@@ -0,0 +1,62 @@
+
+defaults:
+  run:
+    working-directory: supabase/functions
+
+jobs:
+  deno:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    timeout-minutes: 10
+    timeout-minutes: 10
```

actionlint (1.7.12)

[error] key "timeout-minutes" is duplicated in "deno" job. previously defined at line:36,col:5

(syntax-check)